overpass/api.py

```diff
@@ -137,7 +139,17 @@ def get(self, query, responseformat="geojson", verbosity="body", build=True, dat
         return self._as_geojson(response["elements"])
 
     @staticmethod
-    def _api_status() -> dict:
+    def _strptime(date_string):
+        if dependency.Python.less_3_7():
+            dt = datetime.strptime(date_string, '%Y-%m-%dT%H:%M:%SZ')
+            kwargs = {k: getattr(dt, k) for k in ('year', 'month', 'day', 'hour', 'minute', 'second', 'microsecond')}
+            kwargs['tzinfo'] = timezone.utc
+            return datetime(**kwargs)
+
+        return datetime.strptime(date_string, '%Y-%m-%dT%H:%M:%S%z')
+
+    @classmethod
+    def _api_status(cls) -> dict:
```

Comment:
Also deprecated by the dropping of 3.6 support

Reply:
you mean we can get rid of the custom `_strptime()` function altogether and replace with plain `datetime.strptime()`?